robot_manager:
  ros__parameters:
    position_controller_name: "joint_trajectory_controller"
